### PR TITLE
Create API Service App

### PR DESCRIPTION
**Pull request by dzadikdev**
#421 Added Module to Admin App for Creating Service Apps with Client Credentials.

### CODE REVIEW

File: GlobalBehaviorandComponents/templates/manageusers.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
                     <div class="icon-stack icon-stack-lg bg-primary text-white"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-users"><path d="M17 21v-2a4 4 0 0 0-4-4H5a4 4 0 0 0-4 4v2"></path><circle cx="9" cy="7" r="4"></circle><path d="M23 21v-2a4 4 0 0 0-3-3.87"></path><path d="M16 3.13a4 4 0 0 1 0 7.75"></path></svg></div>
                     <div class="ml-3">
                         <h2 class="mb-0">Account</h2>
-                        <p class="lead mb-0">Manage Users</p>
```
Comment (noinarisak) [inline GlobalBehaviorandComponents/templates/manageusers.html:10]: @dzadikdev Shouldn't the `/templates/manageusers.html` file name match to the Apps?.

**Review by noinarisak — COMMENTED** [review]
Let me know about the template name file.

File: GlobalBehaviorandComponents/templates/manageusers.html (hunk 1 above)
Comment (dzadikdev) [reply GlobalBehaviorandComponents/templates/manageusers.html]: Fixed.

Comment (noinarisak) [reply GlobalBehaviorandComponents/templates/manageusers.html]: Thanks @dzadikdev for the update.

**Review by dzadikdev — COMMENTED** [review]
Changes committed for label change.

File: GlobalBehaviorandComponents/templates/components/template-manageapps.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                        <tr>
+                            <th>Name</th>
+                            <th>App Type</th>
+                            <th>View</th>
+                            <th>Delete</th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                        {% if applist|length > 0 %}
+                            {% for i in applist %}
```
Comment (noinarisak) [inline GlobalBehaviorandComponents/templates/components/template-manageapps.html:12]: can you make this ` for app in applist `. Easy to read when I got to access the dict. ie `i['label']` is hard to read.

File: GlobalBehaviorandComponents/templates/components/template-manageapps-createupdate.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
+                    <div id="start">
+                        <table class="table table-striped">
+                        <div class="dropdown">
+                            <button class="btn btn-primary dropdown-toggle" id="dropdownMenuButton" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Application Type</button>
+                            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                                <a class="dropdown-item" href="#!" onclick="show_apiservicecreate();">API Service</a>
+                                <!--<a class="dropdown-item" href="#!">OIDC Application</a>-->
+                                <!--<a class="dropdown-item" href="#!">Native Application</a>-->
```
Comment (noinarisak) [inline GlobalBehaviorandComponents/templates/components/template-manageapps-createupdate.html:8]: Don't need it, delete it!

**Review by noinarisak — CHANGES_REQUESTED** [review]
More request for change.

File: GlobalBehaviorandComponents/templates/components/template-manageapps.html (hunk 2 above)
Comment (noinarisak) [reply GlobalBehaviorandComponents/templates/components/template-manageapps.html:12]: @dzadikdev Is `if applist|length > 0` even required? If `applist` is None would not the `for` loop not even happen.

**Review by noinarisak — COMMENTED** [review]
LGTM

**Review by noinarisak — APPROVED** [review]

**Pull request merged**